rats-apps/src/python/rats/apps/__init__.py

```diff
@@ -48,4 +49,5 @@
     "group",
     "method_service_id",
     "service",
+    "ServiceMethodArg",
```

Comment:
does this have to be exposed?

Reply:
If we want to allow external packages to add decorators that interact with the service decorators, then I think it needs to be exposed.
It's needed in order to type the additional decorators I add in rats-processors (`task` and `pipeline`).